manifests/drupalconsole.pp

```diff
@@ -0,0 +1,22 @@
+class forumone::drupalconsole ($url = 'https://drupalconsole.com/installer')
+{
+  exec { 'forumone::drupalconsole::download':
+    command => "wget ${url} -O /usr/local/bin/drupal",
+    path    => '/usr/bin',
+    creates => '/usr/local/bin/drupal',
+    }
+
+  exec { 'forumone::drupalconsole::drupalinit':
```

Comment:
Just changed this to run as vagrant w/ its environment.  I feel like I should also include a check to make sure /usr/bin/php is available, or maybe add a require => Class[forumone::php]